lib/veres-one-did-doc.js

```diff
@@ -0,0 +1,452 @@
+/*!
+ * Copyright (c) 2018 Veres One Project. All rights reserved.
+ */
+'use strict';
+
+const constants = require('./constants');
+const {LDKeyPair} = require('crypto-ld');
+
+const uuid = require('uuid/v4');
+const jsonpatch = require('fast-json-patch');
+
+const VERES_DID_REGEX = /^(did\:v1\:)(test\:)?(uuid|nym)\:(.+)/;
+
+class VeresOneDidDoc {
+  constructor(options = {}) {
+    this.injector = options.injector;
+
+    this.doc = options.doc || {'@context': constants.VERES_ONE_V1_CONTEXT};
```

Comment:
We're going to have to fix this in the *very* near future, but doesn't have to be in this rev. The `@context` needs to start with the DID context and then use the Veres One context after that (in the very near future).

Reply:
@dlongley so, what will it look like?

Reply:
did doc:

```js
{
  "@context": ["https://w3id.org/did/v1", "https://w3id.org/veres-one/v1"],
  ...
}
```

The above approach allows applications that don't do JSON-LD processing to do a simple check for the presence of `https://w3id.org/did/v1` in the first position of the `@context` array -- and then use whatever general DID terms they want in their app (ignoring any veres-one specific ones, as the rule is that any context that follows the DID one may not redefine terms).

Reply:
@dmitrizagidulin this is copy/paste from a comment on this topic made elsewhere: https://gist.github.com/mattcollier/0c922b1e9550bf7d2e470448516f10cb

Reply:
K, updated the default context to the array above.